packages/wallets/src/features/cashier/modules/Transfer/helpers/helpers.ts

```diff
@@ -0,0 +1,50 @@
+import { THooks, TMarketTypes } from '../../../../../types';
+
+type TGetAccountNameProps = {
+    //@ts-expect-error provide proper type for accounts from transfer_between_accounts response
+    accountCategory: THooks.TransferAccount['account_category'];
+    accountType: THooks.TransferAccount['account_type'] | 'standard';
+    displayCurrencyCode?: THooks.CurrencyConfig['display_code'];
+    mt5MarketType: TMarketTypes.SortedMT5Accounts;
+};
+
+export const getAccountName = ({
+    accountCategory,
+    accountType,
+    displayCurrencyCode,
+    mt5MarketType,
+}: TGetAccountNameProps) => {
+    switch (accountCategory) {
```

Comment:
I know its not yet time for this, but in future, I would love to see all strings in translation files, accessible by key, e.g.

```
{
   "account-name.wallet.GBP": "GBP Wallet"
   "account-name.trading.binary": "Deriv Apps",
   "account-name.trading.dxtrate": "Deriv X"
   ... // all other ones
   }

Reply:
so then no one needs to construct a structure of switches just to translate account name

Reply:
yes @wojciech-deriv 
actually i'm also not happy with this switches.
definitely we should have smth like you mentioned above
this more performant then switches